config/enrichments/currency_conversion_config.json

```diff
@@ -11,6 +10,50 @@
 			"apiKey": "{{KEY}}",
 			"baseCurrency": "USD",
 			"rateAt": "EOD_PRIOR"
+		},
+		"mappings": [
+		{
+			"input": "iglu:ecommerce.tracking/product_entity/jsonschema/1-0-0",
```

Comment:
Do we need a field here to say whether to look in the contexts vs in the unstruct_event?

Reply:
Yes - I think that'd be a good call.

Reply:
Something like `"entity": "event"` vs `"entity": "context"`?  I hate naming things!

Reply:
[This](https://github.com/snowplow/enrich/blob/master/config/enrichments/sql_query_enrichment_config.json#L20) is how it's done in other enrichments

Reply:
Oh, I should have checked what we do in other places.  I'll update this to be consistent with the other configs.  Thanks!